src/Adapters/BlobAdapter.ts

```diff
@@ -67,21 +67,25 @@ export default class BlobAdapter implements IBlobAdapter {
     getBlobContainerURL() {
         return this.storageAccountHostName && this.containerName
             ? `https://${this.storageAccountHostName}/${this.containerName}`
-            : '';
+            : null;
     }
 
     getStorageAccountURL() {
         return this.storageAccountHostName
             ? `https://${this.storageAccountHostName}`
-            : '';
+            : null;
     }
 
     generateBlobUrl(path: string) {
         if (this.useBlobProxy) {
             return `${this.blobProxyServerPath}${path}`;
         } else {
-            // Need to have this done with only storage account since path will always include container name if required
-            return `${this.getStorageAccountURL()}${path}`;
+            if (this.getStorageAccountURL()) {
+                // Need to have this done with only storage account since path will always include container name if required
+                return `${this.getStorageAccountURL()}${path}`;
+            } else {
+                return null;
```

Comment:
Does the request fail if this happens?

Reply:
yes, this is why added null instead of empty string which it was used to. remember, even it is empty string axios makes the call against the window's location. This way, the axis will fail since the url is null.

Reply:
Alright, but how will the user fix this then? Will they be prompted to select a new container or something?

Reply:
this was going to be fixed when the deeplink state picks the url from localstorage. but if there is no url in the local storage either, let me check the scenario not to block the UI to pick the environment configurations.

Reply:
okay, so I am now making getCorsProperties call only if the deeplink context has storage url value (used to still do it and axios was adding window location but throwing 500 error which was out of our interest to surface in the UI - not CORS related error). Not I just changed it so that we dont make get cors properties call unncessarily. In this way, we let the app continue its flow as usual, meaning if there is no storage url or adt url in the deeplink context, we show "Configure environment" error in the UI.

Reply:
So, yes user is prompted to select new container if it is not present in the deeplink context.